easybuild/tools/parallelbuild.py

```diff
@@ -68,7 +68,7 @@ def build_easyconfigs_in_parallel(build_command, easyconfigs, output_dir='easybu
     :param output_dir: output directory
     :param prepare_first: prepare by runnning fetch step first for each easyconfig
     """
-    _log.info("going to build these easyconfigs in parallel: %s", easyconfigs)
+    _log.info("going to build these easyconfigs in parallel: %s", (ec['full_mod_name'] for ec in easyconfigs))
```

Comment:
@Flamefire does this generator get iterated here?

e.g.
```python
import logging
logging.basicConfig(level=logging.INFO)
logging.info("%s", (i for i in [1,2,3]))
logging.info("%s", [i for i in [1,2,3]])
```
returns
```
INFO:root:<generator object <genexpr> at 0x7fc6b37cf550>
INFO:root:[1, 2, 3]
```

Reply:
```suggestion
    _log.info("going to build these easyconfigs in parallel: %s", [ec['full_mod_name'] for ec in easyconfigs])
```

In my test it did. But you are right: Better safe than sorry in case anything changes later on.